docs/en/stack/data-frames/dataframe-limitations.asciidoc

```diff
@@ -0,0 +1,48 @@
+[[ml-df-limitations]]
+== {dataframe-cap} limitations
+
+beta[]
+
+The following limitations and known problems apply to the 7.2 release of 
+the Elastic {dataframe} feature:
+
+[float]
+[[df-limitations-wizard]]
+=== {dataframe-cap} wizard limitation
+
+The {dataframe} wizard in {kib} cannot display the value of a filed selected by 
+using the source index table on the right side if the selected field contains an 
+array.
```

Comment:
This is quite a long sentence.  Could we make it easier to understand by doing something like this?:

If you have fields that contain arrays, you cannot select these fields in the source index table in the Kibana data frame wizard.

Reply:
Should we add something like this too?:

> You can, however, use those fields when you group and aggregate your data in the wizard.